Examples descriptions lang rules (5b7ae0 / bf051a / b5c3f8) (#1073)
act-rules/act-rules.github.io: _rules/html-page-lang-b5c3f8.md

#### Failed Example 1

> This `html` element does not have a `lang` attribute.

Reviewer: Instead of saying what it doesn't have, should we say what it should have? For example:

```suggestion
This `html` element should have a `lang` attribute.
```

Author: That wouldn't match the "[meets / does not meets condition]" pattern that we suggest in #834, I think.

```html
<html></html>
```

#### Failed Example 2

> This `html` element has a `lang` attribute with an empty (`""`) value.

Reviewer: ```suggestion
This `html` element should not have a `lang` attribute with an empty (`""`) value.
```

Reviewer 2: I think this is a good example of why "should" is maybe not as clear. Is this failing because the attribute is empty or because it should have an empty attribute.

It also could lead to double negative confusion, and Jey indicated it could be an issue for unit test stuff.

```html
<html lang=""></html>
```

#### Failed Example 3

> This `html` element has a `lang` attribute whose value is only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).

Reviewer: ```suggestion
This `html` element should not have a `lang` attribute whose value is only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).
```

```html
<html lang=" "></html>
```

#### Failed Example 4

> This `html` element has no `lang` attribute, only a `xml:lang` attribute.

Reviewer: ```suggestion
This `html` element should have a `lang` attribute that matches the `xml:lang` attribute.
```
